nncf/ptq/api/quantization.py

```diff
@@ -0,0 +1,58 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from typing import Optional
+
+from nncf.api.compression import ModelType
+from nncf.ptq.backend import determine_backend
+from nncf.ptq.backend import BackendType
+from nncf.ptq.data.dataloader import NNCFDataLoader
+
+
+def quantize(model: ModelType,
+             calibration_dataset: NNCFDataLoader,
+             preset: str = 'performance',
+             target_device: str = 'ANY',
+             subset_size: int = 300,
+             fast_error_correction: bool = True,
+             model_type: Optional[str] = None):
```

Comment:
Return statement would be helpful. Otherwise, the user may think that the original model is overridden by quantization.

Reply:
Thanks! That's a good point. I fixed it.